docs/source/en/model_doc/bark.md

```diff
@@ -26,8 +26,53 @@ Bark is made of 4 main models:
 
 It should be noted that each of the first three modules can support conditional speaker embeddings to condition the output sound according to specific predefined voice.
 
+### Optimizing Bark
```

Comment:
Some of the inference optimization techniques are described in the [Efficient Inference on a Single GPU](https://huggingface.co/docs/transformers/perf_infer_gpu_one). Perhaps we should link this doc somewhere here?